new/BLorient13156.xml

```diff
@@ -0,0 +1,154 @@
+                                <locus target="#95v"/>
+                                <desc>Ezechiel the Prophet</desc>
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d5">
+                                <locus target="#112v"/>
+                                <desc>ʾAbbā Kiros</desc>
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d6">
+                                <locus target="#127v"/>
+                                <desc>St Cyricus</desc>
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d7">
+                                <locus target="#132v"/>
+                                <desc>Gabriel the Archangel</desc>
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d8">
+                                <locus target="#144v"/>
+                                <desc>St Onuphrius</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <bindingDesc>
+                            <binding contemporary="false" xml:id="binding">
+                                <decoNote xml:id="b1"/>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                                    <material key="leather"/>
+                                </decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <provenance>This manuscript came to Great Britain via Italy, as shown by notes written in what <persName ref="PRS8999Strelcyn"/>
+                            calls "Italianate Latin" (cf. inside of front board and fols 1r, 4r, 160r, and 161r). They indicate that the manuscript was in <placeName ref="LOC4963Naples"/> in 1618.</provenance>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary"/>
+                                            <citedRange unit="page">53-55</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <t:profileDesc xmlns:t="http://www.tei-c.org/ns/1.0">
```

Comment:
From here on, element names are preceded by t:. This is a technical error that happens occasionally, but needs to be corrected manually